src/Serilog/Formatting/Display/MessageTemplateTextFormatter.cs

```diff
@@ -118,7 +118,13 @@ public void Format(LogEvent logEvent, TextWriter output)
                 }
 
                 if (pt.Alignment.HasValue)
+                {
+#if FEATURE_WRITE_STRINGBUILDER
+                    Padding.Apply(output, ((StringWriter)writer).GetStringBuilder(), pt.Alignment);
```

Comment:
nit: but i prefer to minimise whats inside the `#if`. so i would assign `(StringWriter)writer` to a variable outside the `#if`